demisto_sdk/commands/validate/validators/GF_validators/GF101_generic_field_id_prefix_validate.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.generic_field import GenericField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+GENERIC_FIELD_ID_PREFIX = "generic_"
+ContentTypes = GenericField
+
+
+class GenericFieldIdPrefixValidateValidator(BaseValidator[ContentTypes]):
+    error_code = "GF101"
+    rationale = "Required by the platform."
+    description = "Checks if `id` field include prefix `generic_`"
+    error_message = "ID `{generic_id}` is not valid, it should start with the prefix `{generic_id_prefix}`."
+    fix_message = "Change the value of `id` field to `{generic_id}`."
```

Comment:
```suggestion
    fix_message = "Changed the value of `id` field to `{generic_id_prefix}_{generic_id}`."
```
right?

Reply:
@YuvHayun asked to download the fix option
so deleted completely